packages/cactus-plugin-bungee/src/main/typescript/public-api.ts

```diff
@@ -0,0 +1,10 @@
+import { IPluginFactoryOptions } from "@hyperledger/cactus-core-api";
+import { PluginFactoryBUNGEE } from "./plugin-factory-bungee";
+
+export { PluginBUNGEE, IPluginBUNGEEOptions } from "./plugin-bungee";
+
+export async function createBesuPluginFactory(
```

Comment:
@eduv09 I'm guessing this factory function got copied over from the besu connector, which is totally cool (I also copy paste older plugins when writing a new one) but please make sure to rename it to match existing patterns. Let me know if you need any more help on this.

The reason why this is important is because the API server imports only one specific name from plugin packages when it is instantiating the plugin, so it is important to comply with this otherwise the API server won't be able to load your plugin implementation at all.